src/frontends/tensorflow/include/openvino/frontend/tensorflow/exception.hpp

```diff
@@ -6,29 +6,18 @@
 #include "openvino/frontend/exception.hpp"
 
 namespace ov {
-namespace frontend {
-namespace tensorflow {
-
-class NodeContext;
-
-class OpValidationFailure : public ov::frontend::OpValidationFailure {
```

Comment:
Can we check that OVTF doesn't use removed API?

Reply:
It doesn't. Searched here 
https://github.com/openvinotoolkit/openvino_tensorflow/search?q=OpValidationFailure